packages/aws.rb

```diff
@@ -3,25 +3,12 @@
 class Aws < Package
   description 'The AWS CLI is an open source tool built on top of the AWS SDK for Python (Boto) that provides commands for interacting with AWS services.'
   homepage 'https://aws.amazon.com/documentation/cli/'
-  version '1.17.13'
+  version '1.17.13-1'
   license 'Apache-2.0'
   compatibility 'all'
   source_url 'https://github.com/aws/aws-cli/archive/1.17.13.tar.gz'
```

Comment:
New version?  See https://github.com/aws/aws-cli/archive/refs/tags/1.19.44.tar.gz.  You also have the option to jump to version 2.  See https://github.com/aws/aws-cli/archive/refs/tags/2.1.34.tar.gz.

Reply:
Upgrade to version 2?